pkg/reconciler/broker/broker.go

```diff
@@ -161,6 +162,13 @@ func (r *Reconciler) reconcile(ctx context.Context, b *v1alpha1.Broker) error {
 		return nil
 	}
 
+	channelResourceInterface := r.DynamicClientSet.Resource(duckroot.KindToResource(b.Spec.ChannelTemplate.GetObjectKind().GroupVersionKind())).Namespace(b.Namespace)
```

Comment:
Maybe use `UnsafeGuessKindToResource` instead of `KindToResource`? Both are bad, maybe `UnsafeGuessKindToResource` is less bad (cover more cases).

Reply:
Done, thanks!